aten/src/ATen/core/function_schema.h

```diff
@@ -146,12 +146,17 @@ struct FunctionSchema {
     return is_varret_;
   }
   bool is_mutable() const {
-    return std::any_of(
-        arguments_.cbegin(), arguments_.cend(), [](const Argument& arg) {
-          const auto& aliasInfo = arg.alias_info();
-          return aliasInfo && aliasInfo.value().isWrite();
-        });
+    return is_custom_op() ||
+        std::any_of(
+            arguments_.cbegin(), arguments_.cend(), [](const Argument& arg) {
+              const auto& aliasInfo = arg.alias_info();
+              return aliasInfo && aliasInfo.value().isWrite();
+            });
   }
+  bool is_custom_op() const {
```

Comment:
For this PR, I think it would be safer to white-list `prim` and `aten` into the stronger alias behavior. I do not like the idea of making a publically-exposed distinction between custom ops and our ops, even for just a few commits.